content/apps/production-ready.md

```diff
@@ -2,54 +2,84 @@
-there any issue with one of the runners your app is on your app continues to work.
+there any issue with one of the platform runners where your app instances are assigned, your app continues to function correctly.
+
+#### How
+* See the [multiple instances]({{< relref "multiple-instances.md" >}}) page.
+
+### Prefer User Provided Services
+Prefer user provided services over environment variables for a secure centralized way to store application credentials and variables. Environment variables are ephemeral to the specific deployment of each application.
+
+#### How
+* Create [user provided services](https://docs.cloudfoundry.org/devguide/services/user-provided.html) with `cf cups` and bind them with `cf bs`. Once you have updated your application to read your stored credentials from the service, update your `manifest.yml` to make it part of your configuration.
+
+### Prefer dedicated over shared services
+Shared services do not have the same constraints as a dedicated service and can be affected by other customers that are performing CPU or memory intensive tasks against that service.
 
-### How
+#### How
+* Use the `cf marketplace` to consider your options and sizes for each service and choose appropriately for the resources your application will need.
 
-See the [multiple instances]({{< relref "multiple-instances.md" >}}) page.
+### Space per environment
+When running an application for development or testing, it is best to have a separate space from your production instance in which to run the application.
 
-## Prefer User Provided Services
+#### How
+* As an Org Manager for your organization, use the `cf create-space` command to create new spaces for each environment
 
-Prefer user provided services over environment variables for a secure centralized way to store application credentials and variables.
+### Health monitoring
+You want to be able to receive alerts about application errors, downtime, and throughput issues.
 
-### How
+#### How
+* NewRelic provides uptime monitoring with "Insights". It is easy to set up and
+receive alerts on a variety of metrics.
 
-Create user provided services with `cf cups` bind them with `cf bs`.
+## Additional practices
+The following practices are very helpful to incorporate into most cloud.gov apps. Check these out and evaluate which ones you need, depending on the needs of your team and users.
 
-## NewRelic (or similar)
+### Zero-Downtime deploy
+Your application should be able to be deployed without generating any downtime. Be aware there are known issues if your application automatically does database migrations when deploying.
 
-You want to be able to receive alerts about application errors and throughput issues.
+#### How
+* Use the [autopilot](https://github.com/concourse/autopilot) Cloud Foundry CLI plugin or the [`cf-blue-green`](https://github.com/18F/cf-blue-green) tool; see instructions in the respective repositories.
```

Comment:
AFAIK, the only reason to use cf-blue-green over autopilot is if the application doesn't have a proper manifest. Since we're already recommending that relevant configuration lives in the manifest, I don't think we need to recommend cf-blue-green. I also think that using `create-app-manifest` during deployment should be avoided if possible and doesn't belong in a best practices guide.